js/src/jsonrpc/generator/build-markdown.js

```diff
@@ -16,25 +16,46 @@
 
 import fs from 'fs';
 import path from 'path';
+import chalk from 'chalk';
 
+import { DUMMY } from '../helpers';
+import { BlockNumber } from '../types';
 import interfaces from '../';
 
-const MARKDOWN = path.join(__dirname, '../../release/interfaces.md');
+const ROOT_DIR = path.join(__dirname, '../docs');
 
-let preamble = '# interfaces\n';
-let markdown = '';
+if (!fs.existsSync(ROOT_DIR)) {
+  fs.mkdirSync(ROOT_DIR);
+}
+
+// Logging helpers
+function info (log) { console.log(chalk.blue(`INFO:\t${log}`)); }
+function warn (log) { console.warn(chalk.yellow(`WARN:\t${log}`)); }
+function error (log) { console.error(chalk.red(`ERROR:\t${log}`)); }
+
+const type2print = new WeakMap();
```

Comment:
Would probably make sense to move this definition (& init) to above the log functions, just so the variables are more-or-less in the same place.